package.json

```diff
@@ -14,77 +14,86 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "lint:dependency": "ember dependency-lint",
     "start": "ember serve",
-    "test": "npm-run-all lint:* test:*",
+    "test": "npm-run-all lint test:*",
     "test:ember": "ember test"
   },
   "dependencies": {
+    "date-fns": "^2.29.2",
     "ember-cli-release": "^1.0.0-beta.2",
-    "ember-fetch": "^8.0.1",
-    "ember-moment": "^8.0.1"
+    "ember-fetch": "^8.1.1"
   },
   "devDependencies": {
-    "@lblod/ember-submission-form-fields": "^0.15.3",
-    "@appuniversum/appuniversum": "^0.0.14",
-    "@appuniversum/ember-appuniversum": "0.1.7",
-    "@ember/optional-features": "^1.3.0",
-    "@ember/render-modifiers": "^1.0.2",
-    "@glimmer/component": "^1.0.1",
-    "@glimmer/tracking": "^1.0.0",
-    "@lblod/ember-mock-login": "^0.4.7",
-    "@lblod/ember-vo-mu-file-upload": "^0.6.0",
-    "@lblod/ember-vo-webuniversum": "^0.22.6",
-    "@lblod/ember-vo-webuniversum-data-table": "^0.2.11",
+    "@appuniversum/ember-appuniversum": "^1.3.0",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/render-modifiers": "^2.0.4",
+    "@ember/test-helpers": "^2.6.0",
+    "@glimmer/component": "^1.0.4",
+    "@glimmer/tracking": "^1.0.4",
+    "@lblod/ember-mock-login": "^0.7.0",
+    "@lblod/ember-submission-form-fields": "^1.9.2",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-auto-import": "^1.6.0",
-    "ember-cli": "~3.20.2",
-    "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^7.21.0",
+    "ember-auto-import": "^2.4.2",
+    "ember-cli": "~3.28.5",
+    "ember-cli-app-version": "^5.0.0",
+    "ember-cli-babel": "^7.26.10",
     "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-dependency-lint": "^2.0.1",
```

Comment:
I would also add the config to not generate any tests since we already run the cli command.

https://github.com/salsify/ember-cli-dependency-lint#lint-tests

Reply:
I don't know what this means. I don't see how this is generating tests.

Reply:
At the moment ember-cli-dependency-lint generates a test for each addon in the app. Since we already run the cli equivalent with the `lint:dependencies` script we actually run the dependency lint step twice (as can be seen in the CI logs: https://rpio-drone.redpencil.io/lblod/frontend-meldingsplichtige-api/29/1/6).

The addon provides an option to disable that which is documented in the link of my previous comment 😄. So just adding that config should be all that's needed.